package.json

```diff
@@ -10,8 +10,8 @@
     "lint:fix": "eslint --fix --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "lint": "eslint --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "format": "npx prettier --write .",
-    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test*' --exclude test/e2e/* --timeout 5000",
-    "test:e2e:node-module": "ts-node test/e2e/e2e-node-module.ts",
+    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test*' --exclude 'test/e2e/*' --timeout 5000",
```

Comment:
Needed to fix this exclude pattern, otherwise the E2E tests would get run when running `npm run test`.